pages/common/gibo.md

```diff
@@ -0,0 +1,23 @@
+# gibo
+
+> Fetch gitignore boilerplates.
+
+- List available boilerplates:
+
+`gibo list`
+
+- Write a boilerplate to stdout:
+
+`gibo dump {{boilerplate}}`
+
+- Write a boilerplate to .gitignore:
+
+`gibo dump {{boilerplate}} >> {{.gitignore}}`
```

Comment:
On some Linux systems, it's an error to have a space after the `>>`.

Reply:
Ah, didn't know that. Would `gibo dump {{boilerplate}} >>{{{.gitignore}}` be correct?

Reply:
If you remove the 3rd `{`, then yep :slightly_smiling_face:

Reply:
Oops. Didn't make it into the commit.